test/WebSites/BasicWebSite/Views/Home/JsonHelperInView.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@model BasicWebSite.Models.Person
+
+<p>
+    Expected: John &lt;b&gt;Smith&lt;/b&gt;
+</p>
+<script type="text/javascript">
+    var person = @Json.Serialize(Model);
```

Comment:
might be worth adding a `JsonHelperWithSettingsInView.cshtml` as well.  that one would use the second `Serialize()` overload.  (definitely not worth adding unit tests of `JsonHelper`)